app/js/components/orderEntry/PastOrders.jsx

```diff
@@ -77,31 +77,36 @@ export class PastOrders extends React.Component {
                   <PastOrder key={pastOrder.uuid} {...pastOrder} />)}
               </tbody>
             </table>
-            <div className="clear">
-              <div className="float-left-padding">
-                {this.props.showResultCount}
-              </div>
-              <div className="dataTables_paginate">
-                <ReactPaginate
-                  pageCount={this.state.pageCount}
-                  pageRangeDisplayed={5}
-                  marginPagesDisplayed={3}
-                  previousLabel="Previous"
-                  nextLabel="Next"
-                  breakClassName="text-align-center"
-                  initialPage={0}
-                  containerClassName="react-paginate-container"
-                  pageLinkClassName="page-link"
-                  activeClassName="active-link"
-                  disabledClassName="active-link"
-                  nextLinkClassName="page-link"
-                  previousLinkClassName="page-link"
-                  onPageChange={this.onPageChange}
-                  forcePage={this.state.pageNumber}
+            {
+              this.props.pageCount === 1 ?
```

Comment:
When doing conditional render and one returns null or undefined,  try using && or the || operators to achieve that. ie in your case  {this.props.pageCount !== 1 && <the thing you want to render if this.props.pageCount !== 1 >}

Reply:
Thanks for your feedback.
Would look into this.